client/src/components/header/style.css

```diff
@@ -0,0 +1,100 @@
+.container {
+  width: 100%;
+  margin: auto;
+  top: 0px;
+  left: 0px;
+
+  height: 60px;
+  background: #ffffff 0% 0% no-repeat padding-box;
+  box-shadow: 0px 5px 10px #00000014;
+  border-radius: 0px 0px 8px 8px;
+  opacity: 1;
+}
+.header-container {
+  display: flex;
+  margin: auto;
+  justify-content: space-between;
+  width: 80%;
+  /* padding: 30px 120px; */
+  align-items: center;
+}
+
+.header__logo {
+  /* width: 100%; */
+}
+.header-left {
+  display: flex;
+  /* margin-top: 2px; */
+
+  /* margin-top: 10px;
+  width: 50%; */
+}
+.header__list ul {
+  list-style-type: none;
+  display: flex;
+  margin-top: 2px;
+
+  /* justify-content: flex-start; */
+}
+.header__list li {
+  margin-right: 20px;
+  color: black;
+  /* letter-spacing: 2px; */
+}
+.header-right {
+  display: flex;
+  /* width: 50%; */
+  border: 2px solid #287b72;
+  color: #ffede3;
+
+  /* justify-content: flex-end; */
+}
+
+/*
+
+.header-right button {
+  border-radius: 50%;
+  margin-right: 5px;
+  border: none;
+  outline: none;
+  cursor: pointer;
+} */
+
+.home__list {
```

Comment:
the hover color for this element (Home) in the header is blue, it will be better to make the hove color #287b72 (main color)